### PR TITLE
partial dashboard documentation here

### PR DESCRIPTION
**Pull request by thekemkid**

### CODE REVIEW

File: dashboard.md [hunk 1]
```diff
@@ -1,32 +1,52 @@
...
-* Integration API article list endpoint URL
-  An endpoint to return the list of articles to import
-* ...
 
 < Here we'll put some more config options when we decide on them >
 < We should put more detail on these options too (types, valid args, whether optional etc).
+
+## Starting a migration
+
+When flyway has been configured with your brand you will be able to start migrations through the dashboard.
```
Comment (srilq) [inline dashboard.md]: Flyway with a capital F

File: dashboard.md [hunk 2]
```diff
@@ -1,32 +1,52 @@
...
+
+[flyway-dashboard.prod.cni.digital/migration](http://flyway-dashboard.prod.cni.digital/migration)
+
+There is two options available when starting a migration:
+
+* Start a full migration.
+  With this option, Flyway will begin iterating through your full paginated API to retrieve articles and migrate them.
```
Comment (srilq) [inline dashboard.md]: Will this line be rendered as a new block if there is no blank line above (or just a single inline space)?

Comment (thekemkid) [reply dashboard.md]: I would expect them to all render as single bullet points, but within the source I found this formatting nicer

File: dashboard.md [hunk 3]
```diff
@@ -1,32 +1,52 @@
...
+
 ## Authentication
 
-< I don't know how we're going to do authentication yet (or if we'll have separate dashboards per brand) >
+The dashboard may require authentication to access it. The technical details of this has yet to be decided and as such cannot be documented right now.
 
 ## Validation
 
+As mentioned in the [integration section](/integration.md), the Flyway dashboard can validate your api responses during development. This will be useful in understanding if Flyway is able to understand the structure of the article that your integration api returns. Simply paste in a JSON representation of an article and the dashboard will validate your data structure.
+
```
Comment (graycodes) [inline dashboard.md]: `s/api/API/`

**Pull request merged**